ot/gromov.py

```diff
@@ -572,6 +574,349 @@ def df(G):
         return log['fgw_dist']
 
 
+def GW_distance_estimation(C1, C2, p, q, loss_fun, T, number_sample=[-1, -1], std=True):
+    r"""
+        Returns an approximation of the gromov-wasserstein cost between (C1,p) and (C2,q)
+        with a fixed transport plan T.
+
+        The function gives an unbiased approximation of the following equation:
+
+        .. math::
+            GW = \sum_{i,j,k,l} L(C1_{i,k},C2_{j,l})*T_{i,j}*T_{k,l}
+
+        Where :
+        - C1 : Metric cost matrix in the source space

```

Comment:
Add anew line below Where, because the list is not detected (see https://739-71472695-gh.circle-artifacts.com/0/dev/gen_modules/ot.gromov.html#ot.gromov.GW_distance_estimation)